Automattic/wp-calypso
# Support Chat: Make it visible when customizer is open

This PR makes the support chat visible when the customizer is open. Both in docked sidebar, and panel mode.

Screenshots:

![screen shot 2016-10-24 at 19 19 55](https://cloud.githubusercontent.com/assets/1204802/19656170/25de817e-9a1f-11e6-969e-6d8563efed3c.png)

![screen shot 2016-10-24 at 19 19 59](https://cloud.githubusercontent.com/assets/1204802/19656172/295a5044-9a1f-11e6-8eaa-a2178ba6c5d6.png)

To test:

- Open support chat
- Open the customizer
- Verify the chat is still there
